spec/stac-geoparquet-spec.md

```diff
@@ -0,0 +1,39 @@
+# STAC GeoParquet Specification
+
+## Overview
+
+This document specifies how to map a set of [STAC Items](https://github.com/radiantearth/stac-spec/tree/v1.0.0/item-spec) into
+[GeoParquet](https://geoparquet.org). It is directly inspired by the [STAC GeoParquet](https://github.com/stac-utils/stac-geoparquet)
+library, but aims to provide guidance for anyone putting STAC data into GeoParquet. 
+
+## Guidelines
+
+Generally most all the fields in a STAC Item should be mapped to a row in GeoParquet. We embrace Parquet structures where possible, mapping
+from JSON into nested structures. We do pull the properties to the top level, so that it is easier to query and use them. The names of the
+most of the fields should be the same in STAC and in GeoParquet.
+
+| Field           | GeoParquet Type    | Required | Details                                            |
+| --------------- | ------------------ | ---------|--------------------------------------------------- |
+| type            | String             | Optional | This is just needed for GeoJSON, so it is optional and not recommended to include in GeoParquet |
+| stac_extensions | List of Strings    | Required | This column is required, but can be blank if no STAC extensions were used |
+| id              | String             | Required | Required, should be unique |
+| geometry        | Binary (WKB)       | Required | For GeoParquet 1.0 this must be well-known Binary. |
+| bbox 	          | List of Decimals   | Required | Can be 4 or 6 decimals, so won't be a fixed size list. |
+| properties      | per field          | Required | Each property should use the relevant Parquet type, and be pulled out of the properties object to be a top-level Parquet field |
```

Comment:
I think it's probably clearer if this isn't part of the table, because this at first glance makes me think there's a column named `properties`

Reply:
Yeah, I wasn't quite sure how to represent it. My original column name in the table 'STAC Field', but then changed it to be generic, since the stac and parquet seemed like they'd always have the same fields. Moving it out of the table is probably the easiest - I was just going through the top level of STAC.